ApiTests/Utils/XpathPatterns.cs

```diff
@@ -1,13 +1,8 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
-namespace TestProject.Utils
+﻿namespace TestProject.Utils
 {
-    internal class XpathPatterns
+    static class XpathPatterns
     {
         public static readonly string preciseTextXpath = "//*[text()='{0}']";
+        public static readonly string typeText = "//input[@type='text']";
```

Comment:
that's just a specific locator, not something we can reuse in all of the pages, so I'd suggest moving it to the page class where we use it